Gems/ROS2/Code/Source/Lidar/LidarRaycaster.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/Math/Transform.h>
+#include <AzCore/Math/Vector3.h>
+#include <AzCore/std/containers/vector.h>
+#include <AzFramework/Physics/PhysicsScene.h>
+
+// TODO - switch to interface
```

Comment:
This TODO should be made into a github issue: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#todo-comments